packages/core/echo/echo-schema/src/mutable/mutable-schema.test.ts

```diff
@@ -91,4 +91,46 @@ describe('dynamic schema', () => {
     });
     expect(getPropertyMetaAnnotation(registered.getProperties()[0], metaNamespace)).to.deep.eq(metaInfo);
   });
+
+  test('updates typename', async ({ expect }) => {
+    // Create schema with some fields and annotations
+    const registered = createMutableSchema(EmptySchemaType);
+    const originalVersion = registered.storedSchema.version;
+    registered.addFields({
+      field1: S.String.pipe(PropertyMeta('test', { maxLength: 10 })),
```

Comment:
name, age might be better to help comprehend.
ideally re-use existing test use cases.
contact, org, etc.